### PR TITLE
Storage Move functionality [stage-4]

### PR DESCRIPTION
**Pull request by alex-deaconu**
Folder selector updates

Filter out selected folders
Allow root folder selection

Linting and beautify changes

Re-enable coveralls

Fix pendingOperations failure

### CODE REVIEW

**Comment by alex-deaconu** [comment]
This is ready for review. I looked/attempted to write e2e tests for this feature, but it requires extra work for testing folder creation & asset cleanup. I propose we add another card for that implementation (and include e2e tests for duplicate as well).

@ezequielc @fjvallarino please review. Thanks!

File: web/scripts/storage/services/svc-file-actions-factory.js [hunk 1]
```diff
@@ -100,47 +96,63 @@ angular.module('risevision.storage.services')
...
+                'restricted-role') >= 0) {
+              $translate('storage-client.access-denied')
+                .then(function (msg) {
+                  pendingOperationsFactory.statusDetails.message = msg;
+                });
+            } else {
+              $translate('storage-client.' + resp.message, {
+                username: resp.userEmail
+              }).then(function (msg) {
+                pendingOperationsFactory.statusDetails.message = msg;
```
Comment (alex-deaconu) [inline web/scripts/storage/services/svc-file-actions-factory.js]: This piece of code requires the String to actually exist, if it doesn't the promise will never return and no message will be shown. I need to switch this to use `$filter`.

**Comment by ezequielc** [comment]
@alex-deaconu I found a small issue: when a move operation fails (I tested moving to the same folder) when I try another move operation, the title of pending operations panel still shows the error message instead of 'Moving...'. Thanks

**Comment by ezequielc** [comment]
@alex-deaconu Also, probably related to your inline comment above, when I try to move and another file already exists with the same name, the title of pending operations becomes empty instead of showing the error message. Thanks

**Comment by alex-deaconu** [comment]
PR is updated. To confirm, I've fixed the issue with pending operations getting stuck on the error message. Also fixed another related issue.

In order to fix the second issue with the error message strings as well as the comment from the i18n PR, I've done some refactoring in common-i18n for Storage. Unfortunately, due to the fact that the Strings are returned from the Server, and used in 2 different error components, I tried a bunch of different things, but I couldn't prevent duplication.

@ezequielc @fjvallarino wondering if you could review the latest commits and let me know if acceptable. Thanks!

File: web/partials/storage/list-view.html [hunk 2]
```diff
@@ -54,7 +54,7 @@
   <tbody class="table-body">
     <tr class="table-body__row"
     ng-click="fileClick(file);"
-    ng-class="{'active': file.isChecked, 'table-body__row--blocked-file': file.isThrottled, 'table-body__row--no-select': !storageFactory.canSelect(file), 'table-body__row--disabled': storageFactory.isDisabled(file) }"
+    ng-class="{'active': file.isChecked, 'table-body__row--error.blocked-file': file.isThrottled, 'table-body__row--no-select': !storageFactory.canSelect(file), 'table-body__row--disabled': storageFactory.isDisabled(file) }"
```
Comment (ezequielc) [inline web/partials/storage/list-view.html]: @alex-deaconu Is this change in the class name intentional?  Thanks

Comment (alex-deaconu) [inline web/partials/storage/list-view.html]: @ezequielc definitely not. Good catch, fixed!

**Comment by ezequielc** [comment]
@alex-deaconu 👍  Thanks for fixing the issues. I had one minor comment, but I saw you are already fixing it. Thanks

**Comment by alex-deaconu** [comment]
@ezequielc @fjvallarino rebased this and https://github.com/Rise-Vision/common-i18n/pull/222 off of the Duplicate changes. This should be good to go, wondering if you could do a final review. Thanks!

File: bower.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
     "angular-mocks": "1.5.0",
     "common-header": "https://github.com/Rise-Vision/common-header.git#v1.13.4",
     "component-storage-selector": "https://github.com/Rise-Vision/component-storage-selector.git",
-    "rv-common-i18n": "https://github.com/Rise-Vision/common-i18n.git#v1.4.18",
+    "rv-common-i18n": "https://github.com/Rise-Vision/common-i18n.git#feature/storage-move",
```
Comment (fjvallarino) [inline bower.json]: @alex-deaconu maybe we should release a new common-i18n version

**Comment by fjvallarino** [comment]
@alex-deaconu looks good! :+1:
